src/components/pacbio/PacbioPoolEdit.vue

```diff
@@ -28,14 +28,10 @@
                 <traction-label>Auto tagging</traction-label>
                 <traction-toggle v-model="autoTag" data-attribute="check-box" />
               </fieldset>
-              <fieldset v-if="!!tubeItem.barcode" class="flex flex-col">
+              <fieldset v-if="!!pool.barcode" class="flex flex-col">
                 <traction-label class="h-full">Pool Barcode</traction-label>
-                <traction-label
-                  v-if="!!tubeItem.barcode"
-                  data-attribute="barcode"
-                  class="font-bold text-nowrap"
-                >
-                  {{ tubeItem.barcode }}
+                <traction-label data-attribute="barcode" class="font-bold text-nowrap">
+                  {{ pool.barcode }}
```

Comment:
I hope v-if is removed because this component will only be displayed from a pool that will definitely have a barcode, so checking for it doesn't make sense.

Reply:
Yes thats right because the above `v-if` dictates whether or not it is visible anyway.